src/plugins/era5/datasource.py

```
@@ -0,0 +1,353 @@
+        **filters: Any
+    ) -> AsyncIterator[List[Measurement]]:
+        if not self.cds_api_key:
+            logger.warning("ERA5 data access requires CDS API key. Visit https://cds.climate.copernicus.eu/")
+            async for measurements in self._get_demo_measurements(sensor, start_date, end_date, limit):
+                yield measurements
+            return
+            
+        try:
+            async for measurements in self._fetch_era5_data(sensor, start_date, end_date, limit):
+                yield measurements
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)